### ISSUE TITLE
Too many open files

### ISSUE THREAD

**Issue by fpietka** [junegunn/fzf#508]
Hi, I currently have an issue preventing `fzf` to read my history:

```
$(__fzf_history__)create pane failed: env TERM=screen-256color  bash /var/folders/n2/hs64sw9j2z7_xnk1_s2bs4ph0000gn/T//fzf-args-1973: Too many open files
```

I might add I'm within a tmux sessions. For now I didn't try to reboot or detach the session.

**Comment by junegunn** [comment]
It's irrelevant to fzf. Check the output of `lsof` and clean up processes with many open files. Or simply reboot.

**Comment by fpietka** [comment]
I'll give more information here, just in case someone step on it.
 
lsof isn't alarming, also I can still open many files, create new tabs on chrome, all that without any problem.
So I tried something. `__fzf_history` is using `__fzf_cmd` which is giving me `fzf-tmux -d40%` (I may add I got the same outside of a tmux session).
So I tried executing the following:
`history | fzf-tmux -d40% +s --tac +m -n2..,.. --tiebreak=index --toggle-sort=ctrl-r`
It gave me the error within the tmux session, but outside of tmux it is working fine.

Then I tried the following which is working fine, even inside a tmux session:
`history | fzf +s --tac +m -n2..,.. --tiebreak=index --toggle-sort=ctrl-r`

So I tried to open new panes, and in fact I had just one pane left to create until tmux gave me the `too many files open` error itself.

I currently have 38 windows in tmux. It seems like a lot, but also isn't that much.
Anyway I'll keep investigating on this tmux issue.

Thanks for your time.
